### ISSUE TITLE
Duplicate `poddefaults` CRD causes the charm to get into error state

### ISSUE THREAD

**Issue by DnPlas** [canonical/kubeflow-profiles-operator#92]
#79 added the `poddefaults` CRD so it gets rendered and applied on deployment; this causes a conflict with `admission-webhook` when deployed alongside `profiles` (e.g. in the kubeflow bundle or integration tests), as both charms try to apply the same k8s resource:

```
unit-kubeflow-profiles-0: 21:52:50 ERROR unit.kubeflow-profiles/0.juju-log Uncaught exception while in charm code:
Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-kubeflow-profiles-0/charm/venv/lightkube/core/generic_client.py", line 188, in raise_for_status
    resp.raise_for_status()
  File "/var/lib/juju/agents/unit-kubeflow-profiles-0/charm/venv/httpx/_models.py", line 745, in raise_for_status
    raise HTTPStatusError(message, request=request, response=self)
httpx.HTTPStatusError: Client error '409 Conflict' for url 'https://10.152.183.1/apis/apiextensions.k8s.io/v1/customresourcedefinitions/poddefaults.kubeflow.org?fieldManager=lightkube'
For more information check: https://httpstatuses.com/409
```

#### Possible fix
#91, particularly [this comment](https://github.com/canonical/kubeflow-profiles-operator/pull/91#issuecomment-1434856806)

**Comment by beliaev-maksim** [comment]
@DnPlas can we close?

**Comment by NohaIhab** [comment]
solved by #91
